docs/explanations/principles.md

```diff
@@ -0,0 +1,106 @@
+
+
+# Gutenberg Principles
+
+This document has been written so that those contributing to the Gutenberg project can have some understanding of what guides decisions around what is included in the project (or not).
+
+An important thing to keep at top of mind when reviewing these principles is that they are not isolated from each other. There is a cohesiveness and complementary nature between them that results in a synergistic impact and direction.
+
+When applied, sometimes these principles build on each other much like the idea of constructing a house with a foundation, walls and roof. However, other times the application of the principles results in something that is more similar to a tapestry where threads of each principle are embedded together and evident in the finished feature or design of the project.
+
+
+## Aim for delightful, consistent, accessible user interfaces for empowering content creation and expression.
+
+_“Optimize for the user” (Matias - [Gutenberg or the Ship of Theseus](https://matiasventura.com/post/gutenberg-or-the-ship-of-theseus/))_
+
+Each of the words in this principle are deliberate:
+
+
+### Delightful
```

Comment:
As much as I love this word, it's just not a good principle because it is so subjective. It has also been confiscated by commercial projects.

Reply:
Is it primarily the specific word you object to or also the meaning behind the choice of the word (and the intent of the principle)? If just the word itself, do you have any suggestions about what could be used instead?

Reply:
Delightful does sound a bit like marketing-speak. It's not so much the subjectivity as the underlying idea that we're somehow directing the user's feelings about the product that makes me uneasy.

The word "transparent" popped into my head when reading `The user interfaces should “get out of the way” and support the user`. Transparency both in the sense of clarity (it's clear how to use the interface, thus "supporting" the user instead of making things difficult) but also in the sense of invisibility (it gets "out of the way" so users don't even notice it while working).

Reply:
You both have shared perspectives on the word "delightful" that I didn't even think of. I was thinking of it in the context of aiming for user experiences that elicit "surprised joy" (my interpretation of "delight"). I also don't think we can escape the reality that we indeed _do_ influence users feelings about the product when building it.  This principle is aimed at thinking how what we build contributes to positive feelings.

With that said, I do see how **transparent** could be a good substitute here for the word delightful and moves more towards the objectiveness of a _principle_ as opposed to the subjectivity of a value or goal.

Reply:
+1 for `transparent` :+1:

Reply:
"Delightful" doesn't capture what I think this section hints at. "Transparent" is also lacking a bit in concrete meaning.

The way I'd describe this is "A canvas for expression" (more on that here: https://github.com/WordPress/gutenberg/issues/18667)

Reply:
I'll see if I can capture some of what you wrote in that issue Matías - ❤️ it.

Reply:
See the changes made in d6db504.  I incorporated much of what Matías articulated in #18667 and this forms the basis for the revised content of the principle which is now simply "A canvas for expression" with the supporting points of this canvas:

- Guiding users around interacting with the content
- Uniformity within diversity
- and Accessbility

The wording around accessibility has also been tweaked in f8a44af which includes not only the work around this for _creators_ of content, but also for _consumers_ of content (to address what was [brought up here](https://github.com/WordPress/gutenberg/pull/30963#discussion_r620308866)).

This also handles the concerns around the word "delightful".